src/util-hugepages.c

```diff
@@ -36,18 +37,28 @@ static void SystemHugepageNodeInfoDestroy(NodeInfo *n);
 static void SystemHugepageNodeInfoDump(NodeInfo *n);
 static void SystemHugepageSnapshotDump(SystemHugepageSnapshot *s);
 
+typedef enum OSHpageAction_ {
```

Comment:
this looks a but hard on my eyes, can we just call with `OSHugepageAction_`?

Reply:
changed